core/templates/app.constants.ts

```diff
@@ -158,10 +158,12 @@ export class AppConstants {
 
   public static ENTITY_TYPE = {
     EXPLORATION: 'exploration',
+    EXPLORATION_SUGGESTION: 'exploration_suggestions',
```

Comment:
Please be consistent with the naming -- use "suggestions" throughout. I'm seeing "suggestion" mixed with "suggestions".

Also I am a bit concerned about this "ENTITY_TYPE". It doesn't really seem like an entity type. Maybe you should call the category for the new fields something like IMAGE_CONTEXT?

Reply:
done